### DISCUSSION TITLE
Lyrics not available anymore ?

### DISCUSSION THREAD

**Discussion by NexusPenguin** [rern/rAudio#1277] [General]
Hi, 

One feature I really like about Rune Audio is it 's ability to display the lyrics. 
I think it's the only one doing it... I haven't seen that feature in moOde or Volumio. 

But right now it doesn't display anything anymore...

If I play really popular songs be it with UPnP or directly from the NAS, it does not display any lyrics. 
And I am pretty sure it used to display almost every lyrics for every song some days (weeks) ago :-(

It does not display anything even though the lyrics are included in metadata. 
It does not display anything even though the lyrics are included in a .lrc file in the same folder. 
It does not display anything enven though the lyrics are included in a .txt file in the same folder. 

Has something changed (maybe the way to access those lyrics...)
Am I doing something wrong ? 

I am using the last avaialble image : with or without update, it shows the same behaviour. 
I have not changed any setting in my router. 

It would be nice to know : 
- how lyrics are retrieved by rAudio ; 
- if they're fetched online : from what site do you get them ; 
- if it is possible to watch a log to see what happens. 

Regards.

**Comment by cmh714** [comment]
I see there is a setting on the Settings page for "Lyrics in File"....I dont use that feature. I do get lyrics on radio stations if i look. GL!

**Reply by cmh714** [reply]
EDIT: NM this...mine is off but still get lyrics for songs on NAS. Very interesting issue for you.

**Reply by NexusPenguin** [reply]
Hi,

Thanks for your answer. 

Here, the setting has no effect. 

It doesn't matter if it is enabled or disabled : I still don't get lyrics :-(

The really weird thing is that until something like 10 days ago, I had lyrics on every song. 
It was only marginally that rAudio could not retrieve lyrics for a song. 

All the sudden, I had no lyrics from files played from NAS via UPnP anymore but still got those when they were in the metadata. 

Now I have NONE : be it played directly from rAudio from NAS or be it sent via a remote UPnP/DLNA remote controller :-(

I'll try to hook up a USB stick to see if it can retrieve from files stored locally... If it does, then something is wrong with network. 

Regards. 

FRED

P.S : in case it can help, the music server is not really a NAS, but a "Mini computer" that is running Arch. But I don't think I have updated that since a rather long time.

**Comment by NexusPenguin** [comment]
Hi, 

Replying to myself because the problem is solved. 

As a lot of people reported taht their "Lyrics" were showing as they should, I wondered why mine did not. 

It was commonly said that Lyrics are retrieved over internet on the fly. 
So this of course supposes that you have an active internet connection. 

I thought that having the RaspBerry Pi4 connected to an Ethernet switch that is itself connected to an internet connected router was enough. 

But in doubt I turned on Wi-Fi and connected it to my local Wi-Fi network. 
And... Voilà. Lyrics again. 

The funny thing is that I got "Bio" of artist. So I assumed that I was connected. 
But it seems not :-(

Anyways : with Wi-Fi turned on, Lyrics are there again... Would be nice to allow even if Wi-Fi is turned off.

**Comment by NexusPenguin** [comment]
Hi,

Today, again, lyrics not available. 
They were available yesterday and I did not touch anything !

What is going on ? 


It would be nice if we could AT LEAST have something that can give a hint about WHY it is not working ? 
That would give a clue where to look at and try to solve the problem. 
Because I assume taht everyone else has lyrics so I would be grateful to know why I do not have them...
An error message or something the like would be nioce :-(

If I click on "BIO" I have the artist BIO so I guess I am connected to internet. 
But it seems the "Lyrics" finder does not find it's way through. 

I've of course rebooted the thing about 15 times...

**Reply by NexusPenguin** [reply]
And today... Lyrics back again. 

Unexplainable :-(

**Reply by NexusPenguin** [reply]
And... Gone again :-(

I've tried to change the Emby server used ports. 
I've tried to activate Wi-Fi. 
I've tried to update rAudio with pacman. 
I've tried to enable Lyrics in file. 

What else can I try ? 

Anyways, I think for now, until this has a rational explanation, I'll listen to symphonies and instrumental music :-)

**Comment by NexusPenguin** [comment]
Hi, 

Anyone with an idea about why my lyrics are not showing ? 

Regards.

**Comment by clriis** [comment]
Even here. Lyrics not showing anymore. 

Only get this popup:

![image](https://github.com/rern/rAudio/assets/16999072/b6465f03-b192-4a63-aaa8-84958953343b)

**Reply by NexusPenguin** [reply]
Hi,

Glad to hear I am not the only one. 

Maybe we should try to find if there is something in common in our set-up ? 

Regards. 

FRED

**Reply by Roland-B** [reply]
As @rern doesn't seem to reply to a great deal of users efforts to report bugs (or reply in general) I'll have a go although I may be wrong.

Looks as if he has removed the old functionality and made it so you have to configure yourself, but practially made it useless. Go to Features page and enable lyrics and you will see a setup option. There are no clues to where you can find lyrics sites with start tags and end tags to put in. If anyone manages to find a setting that works please post back here.

Roland.

**Comment by 010x010** [comment]
![0](https://github.com/rern/rAudio/assets/135772339/92b83c4a-24ff-4682-af40-81bd2e20566b)

**Reply by Roland-B** [reply]
@010x010 Great, thanks!

**Reply by cYCL157** [reply]
thanks!  this helped me -- it's bugged me that this has not worked for awhile, but clearly not enough to really dig into the discussions to see if there was something I missed....and I did.
